
syntax = "proto3";
package mam;

message LoadResult {
    map <string, AccountOverviews> map_account_overview = 1;
    map <string, ProfitDetails> map_profit_detail = 2;
}

message AccountOverviews {
    repeated AccountOverview content = 1;
}

message ProfitDetails {
    repeated ProfitDetail content = 1;
}

message AccountOverview {
    int32 accountid = 1;
    double turnover = 2;
    double rate = 3;
    double fee = 4;
    double profit = 5;
}

message ProfitDetail {
    int32 accountid = 1;
    string symbol = 2;
    int32 cfgvol = 3;
    int32 buyvol = 4;
    int32 sellvol = 5;
}
